doc/contribute/guidelines.rst

```diff
@@ -457,8 +457,8 @@ exceptions:
   comment, ``//``, is not allowed.
 * Use ``/**  */`` for doxygen comments that need to appear in the documentation.
 
-When there are differences between the guidelines above and the rules enforced
-by the ``clang-format`` configuration, the guidelines above take precedence.
+When there are differences between the guidelines above and the formatting
```

Comment:
Looks like this change belongs in the previous commit, also does not match the commit description.

Reply:
Good catch. Fixed.